modin/engines/base/frame/partition.py

```diff
@@ -130,6 +133,17 @@ def to_numpy(self, **kwargs):
         """
         pass
 
+    def __copy__(self):
```

Comment:
`__copy__` method was missing in the base partition class. Suggested that it wasn't on purpose and added this to be able to use `copy` in the base implementation of the `mask` method.